site/pages/contact.tsx

```diff
@@ -13,32 +14,38 @@ const Contact = (): ReactElement => {
                     <h2 className="govuk-heading-m">Feedback and support</h2>
                     <p className="govuk-body">
-                        If you are experiencing technical issues, or if you have any suggestions, comments or criticisms, please contact the Create Transport Disruption Data team through one of the channels below.
+                        If you are experiencing technical issues, or if you have any suggestions, comments or
+                        criticisms, please contact the Create Transport Disruption Data team through one of the channels
+                        below.
                     </p>
                     <p className="govuk-body">
-                        The Help Desk is available Monday to Friday, 9am to 5pm (excluding Bank Holidays in England and Wales, and the 24th of December).
+                        The Help Desk is available Monday to Friday, 9am to 5pm (excluding Bank Holidays in England and
+                        Wales, and the 24th of December).
                     </p>
                     <p className="govuk-body">The Help Desk can be contacted by telephone or email as follows.</p>
                     <p className="govuk-body">
                         Telephone: {SUPPORT_PHONE_NUMBER}
                         <br />
-                        Email: <a className="govuk-link" href={`mailto:${SUPPORT_EMAIL_ADDRESS}`}>{SUPPORT_EMAIL_ADDRESS}</a>
+                        Email:{" "}
+                        <Link className="govuk-link" href={`mailto:${SUPPORT_EMAIL_ADDRESS}`}>
+                            {SUPPORT_EMAIL_ADDRESS}
+                        </Link>
                     </p>
                     <h3 className="govuk-heading-s">Related services</h3>
                     <p className="govuk-body">
                         If your query relates to the use of the Bus Open Data Service go&nbsp;
-                        <a href="/contact" aria-label="go to the bus open data service" className="underline">
+                        <Link href="/contact" aria-label="go to the bus open data service" className="underline">
                             here
-                        </a>
+                        </Link>
                         &nbsp;to view their contact details
                     </p>
                 </div>
                 <div className="govuk-grid-column-one-third">
                     <h2 className="govuk-heading-s">Create Transport Disruption Data Service</h2>
                     <p className="govuk-body">
-                        Create Transport Disruption Data
-                        The Create Transport Disruption Data gives local transport authorities and operators within England the ability to create disruptions for bus, light rail, ferry and tram services
-                        Help documents
+                        Create Transport Disruption Data The Create Transport Disruption Data gives local transport
```

Comment:
Is this correct? Looks a little strange

Reply:
Yeah there was some unnecessary text at the start there, I've removed it